tests/system/providers/google/cloud/cloud_build/example_cloud_build_trigger.py

```diff
@@ -42,29 +47,32 @@
 ENV_ID = os.environ.get("SYSTEM_TESTS_ENV_ID")
 PROJECT_ID = os.environ.get("SYSTEM_TESTS_GCP_PROJECT") or DEFAULT_GCP_SYSTEM_TEST_PROJECT_ID
 
-DAG_ID = "example_gcp_cloud_build_trigger"
+DAG_ID = "gcp_cloud_build_trigger"
 
 # Repository with this name is expected created within the project $SYSTEM_TESTS_GCP_PROJECT
 # If you'd like to run this system test locally, please
 #   1. Create Cloud Source Repository
 #   2. Push into a master branch the following file:
 #   tests/system/providers/google/cloud/cloud_build/resources/example_cloud_build.yaml
-GCP_SOURCE_REPOSITORY_NAME = "test-cloud-build-repo"
+GCP_SOURCE_REPOSITORY_NAME = "test-cloud-build-repository"
 
-TRIGGER_NAME = f"cloud-build-trigger-{ENV_ID}"
+TRIGGER_NAME = f"cloud-build-trigger-{ENV_ID}".replace("_", "-")
+PROJECT_NUMBER = "{{ task_instance.xcom_pull('get_project_number') }}"
 
 # [START howto_operator_gcp_create_build_trigger_body]
 create_build_trigger_body = {
     "name": TRIGGER_NAME,
     "trigger_template": {
         "project_id": PROJECT_ID,
         "repo_name": GCP_SOURCE_REPOSITORY_NAME,
-        "branch_name": "main",
+        "branch_name": "master",
     },
     "filename": "example_cloud_build.yaml",
+    "service_account": f"projects/{PROJECT_ID}/serviceAccounts/{PROJECT_NUMBER}-compute@developer.gserviceaccount.com",
 }
 # [END howto_operator_gcp_create_build_trigger_body]
 
+# SERVICE_ACCOUNT = "{{ task_instance.xcom_pull('get_project_number') }}"
```

Comment:
@VladaZakharova what is the reason for this comment or it's just a typo?

Reply:
typo 🙃